src/main/java/com/google/api/codegen/config/PackagingConfig.java

```diff
@@ -104,7 +107,7 @@ private static PackagingConfig createFromString(String yamlContents) {
             .apiName((String) configMap.get("api_name"))
             .apiVersion((String) configMap.get("api_version"))
             .organizationName((String) configMap.get("organization_name"))
-            .protoPackageDependencies((List<String>) configMap.get("proto_deps"))
+            .protoPackageDependencies(nullToEmpty((List<String>) configMap.get("proto_deps")))
```

Comment:
How about using Objects.firstNotNull here?

Reply:
ok I can do that